pkg/providers/bitwarden/bitwarden_test.go

```diff
@@ -0,0 +1,91 @@
+package bitwarden
+
+import (
+	"os"
+	"testing"
+)
+
+func TestGetAddressConfig(t *testing.T) {
+	// Test case 1: cfgAddress is not empty
+	cfgAddress := "http://example.com"
+	expected := "http://example.com"
+	result := getAddressConfig(cfgAddress)
+	if result != expected {
+		t.Errorf("Expected %s, but got %s", expected, result)
+	}
+
+	// Test case 2: cfgAddress is empty, but envAddr is not empty
+	os.Setenv("BW_API_ADDR", "http://env.example.com")
+	expected = "http://env.example.com"
+	result = getAddressConfig("")
+	if result != expected {
+		t.Errorf("Expected %s, but got %s", expected, result)
+	}
+
+	// Test case 3: cfgAddress and envAddr are empty
+	os.Setenv("BW_API_ADDR", "")
+	expected = "http://localhost:8087"
+	result = getAddressConfig("")
+	if result != expected {
+		t.Errorf("Expected %s, but got %s", expected, result)
+	}
```

Comment:
unset env

Reply:
@smerschjohann then I will merge this. thanks so much.

Reply:
@yxxhero done